newlib/libc/include/sys/features.h

```diff
@@ -39,6 +39,7 @@ extern "C" {
 /* Version with trailing underscores for BSD compatibility. */
 #define	__GNUC_PREREQ__(ma, mi)	__GNUC_PREREQ(ma, mi)
 
+
```

Comment:
careful to avoid adding unnecessary whitespace chages to patches

Reply:
That's the empty line which I previously removed with bf9ce95 ;)